spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -716,7 +716,7 @@ class CometSparkSessionExtensions
       // enabled.
       if (isANSIEnabled(conf)) {
         if (COMET_ANSI_MODE_ENABLED.get()) {
-          logWarning("Using Comet's experimental support for ANSI mode.")
+          logInfo("Using Comet's experimental support for ANSI mode.")
```

Comment:
Is this change necessary for Spark 4 support?

Reply:
Without this change, the log gets really long and makes things hard to follow...